src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => century
+    ? (person.sex === 'm' && Math.ceil(person.died / 100) === century)
+    : person.sex === 'm'
+  );
+
+  const menAges = men.map(man => (man.died - man.born));
+
+  const sumAges = menAges.reduce((sum, age) => sum + age, 0);
+
+  return sumAges / menAges.length;
```

Comment:
```suggestion
const menAges = men.reduce((acc, man) => acc + (man.died - man.born);

return menAges / men.length ;
```

Reply:
@Yevhenii0536 Good advice. But why you've approved PR?

Reply:
@sergii-nosachenko , sorry, because i thought that this is a little mistakes